Request/AbstractRequestBodyParamConverter.php

```diff
@@ -151,21 +155,23 @@ protected function execute(Request $request, ParamConverter $configuration)
     }
 
     /**
-     * @return DeserializationContext
+     * @return DeserializationContext|Context
      */
     protected function getDeserializationContext()
```

Comment:
These functions allow to define the (de)serialization context for a specific view. How would we do this with only the serializer api?